src/base/bittorrent/session.cpp

```diff
@@ -3479,6 +3482,21 @@ void Session::setMaxRatioAction(MaxRatioAction act)
     m_maxRatioAction = static_cast<int>(act);
 }
 
+int Session::AIOThreads() const
+{
+    int num = m_AIOThreads;
+    return qMax(num, 4);
```

Comment:
Is there a reason to have 4 as a low boundary? I mean if possible, the boundary should be at 1.
Also please merge the 2 statements.

Reply:
1) The default value in libtorrent settings is 4
Ref: https://libtorrent.org/reference-Settings.html#aio_threads

2) ok, wait a minute

Reply:
I know, but it is a tunable parameter, we shouldn't decide what is best for users unless the value (1~3) is not usable.

Reply:
ok, maybe set the minimum to 1 and the default value to 4?

Reply:
>ok, maybe set the minimum to 1 and the default value to 4?

Yes!